internal/v4/api_test.go

```diff
@@ -78,7 +79,7 @@ func (s *APISuite) SetUpTest(c *gc.C) {
 
 func newServer(c *gc.C, session *mgo.Session, si *charmstore.SearchIndex, config charmstore.ServerParams) (http.Handler, *charmstore.Store) {
 	db := session.DB("charmstore")
-	store, err := charmstore.NewStore(db, si, nil)
+	store, err := charmstore.NewStore(db, si, &bakery.NewServiceParams{})
```

Comment:
If the pointer is nil then a bakery is not created, where as if it is empty parameters it is created with default configuration. My tests require the latter.